nucleus/cluster/ssh/src/main/java/org/glassfish/cluster/ssh/launcher/JulJschLogger.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0, which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception, which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ */
+package org.glassfish.cluster.ssh.launcher;
+
+import com.jcraft.jsch.JSch;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class JulJschLogger implements com.jcraft.jsch.Logger {
+
+    private final Logger logger;
+
+    public JulJschLogger() {
+        this.logger = Logger.getLogger(JSch.class.getName());
```

Comment:
It would be better to use System.Logger instead of direct JUL usage. We are slowly moving in this direction.

Reply:
Good catch. I forgot about this. I switched to System.Logger for Jsch. But I kept the JUL logger in the SSHLauncher which was already there.